components/renku_data_services/data_connectors/api.spec.yaml

```diff
@@ -683,6 +683,9 @@ components:
               provider:
                 type: string
                 description: The provider this value is applicable for. Empty if valid for all providers.
+            required:
+              - value
+              - help
```

Comment:
question: This and the other change in the apispec.yaml are breaking changes for API. Now this by itself is usually not a problem since we do these often from time to time. But in this case we use this API to validate responses (i.e. when we list data connectors I think or when we list the config for a data connector). So it can happen that we have stuff in the DB that was validated against the old API that breaks this API. 

So I think that we either try to do a migration for this. OR we simply remove the validation on listing things - as long as the ui can handle the uncertainty this may represent (or the ui does not touch these properties that much).

Reply:
These fields ARE NOT coming from the DB but from the rclone schema. As such, they should be following the rclone schema. The `sensitive_fields` are copied from the schema by matching the corresponding fields, so this will pass validation (the schema endpoint passes validation).